pkg/terminal/terminal.go

```diff
@@ -81,14 +100,20 @@ func New(client service.Client, conf *config.Config) *Term {
 		client.SetReturnValuesLoadConfig(&LongLoadConfig)
 	}
 
+	if conf.SourceListLineColor < ansiBlack ||
+	conf.SourceListLineColor > ansiBrWhite {
```

Comment:
You need to run `go fmt pkg/terminal/terminal.go`, also I'd only check `if conf.SourceListLineColor == 0`.

Reply:
OK, thanks forgot a final fmt check there.
Are you certain we should only check against 0 though? With no range check, a user could set

source-list-line-color: -1

resulting in an invalid ANSI escape sequence in source list, such as

[-1m    22:		MOVQ	$0, R10
[-1m    23:		MOVQ	$0, R8

..dumb for the user to do that, but I just tested and that's what would occur.